dgraph/cmd/alpha/run.go

```diff
@@ -108,11 +108,12 @@ they form a Raft group and provide synchronous replication.
 	flag.StringP("postings", "p", "p", "Directory to store posting lists.")
 	flag.String("tmp", "t", "Directory to store temporary buffers.")
 
-	// Options around how to set up Badger.
-	flag.String("badger.compression", "snappy",
-		"[none, zstd:level, snappy] Specifies the compression algorithm and the compression"+
-			"level (if applicable) for the postings directory. none would disable compression,"+
-			" while zstd:1 would set zstd compression at level 1.")
+	flag.String("badger", worker.BadgerDefaults,
+		`Various badger options.
+	goroutines=N provides the number of goroutines to use in badger.Stream.
+	compression=[none, zstd:level, snappy] specifies the compression algorithm and the compression 
```

Comment:
We need this `compression` flag in Alpha for [setting the compression type](https://github.com/dgraph-io/dgraph/pull/7387/files#diff-a72445362e52218163e2d2f572b946ff3aea61f8675aa235a5dab3227ac6168eR610) of Badger.